src/components/calcite-radio-group/calcite-radio-group.e2e.ts

```diff
@@ -24,7 +24,8 @@ describe("calcite-radio-group", () => {
     expect(selected).not.toBeDefined();
   });
 
-  it("when multiple items are checked, last one wins", async () => {
+  // skipped. always failing
```

Comment:
[Nitpick] I've created an issue for this (https://github.com/Esri/calcite-components/issues/185), so this comment can be dropped.